scripts/README.md

```diff
@@ -0,0 +1,64 @@
+Publish Metric Scripts
+==========================
+
+These scripts are designed to analyse collection data over time and provide insights into how and what we are publishing.
+
+Publish-log-analysis provides breakdowns of content types and volumes week by week.
+
+Publishing-performance-analysis looks at the start and end times of publish events compared with the amount of data going out, to measure 9.30 efficiency.
+
+How to run the scripts
+------
+
+### Setup the data
+Update the /publish-archive on the publishing mount to include all collection data you wish to analyse
+
+* You can do stuff like `sudo cp -r 2018-07-1* /publish-archive/` to do it by 10 day batches but even that uses a fair amount of CPU and is quite slow. Much better to do it late on when publishing aren’t around
+
+You can manually create a snapshot of the /publish-archive volume in AWS, and then create a new volume to mount to a new machine. This means running the script and chomping CPU won’t get in the way of live systems. Or you can run the scripts out of hours.
+
+### Run the Go script to get the data
+
+1. Checkout this repo locally.
+
+2. Build the binary for whichever script you're running: 
+`GOOS=linux GOARCH=amd64 go build -o metrics publish-log-analysis/main.go`
```

Comment:
Would it be worth creating a Makefile for this? (just a suggestion, not a change request)

Reply:
Thats a good idea, I've added one for now, we might well iterate on it if we look to automate these in future :)